core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -311,8 +309,8 @@ export class HorizontalBarChart extends Chart {
       .attr("class", "bar-group");
 
     barGroups.exit().remove();
-    this._displayBars(barGroups, enteringBarGroups);
-    if (this._barValueColor) this._displayValues(barGroups, enteringBarGroups);
+    this._displayBars(enteringBarGroups);
```

Comment:
And now barGroups is only used here. Do you actually need a variable for that?

Reply:
is used in alignBarText(barGroups) as well, got deleted by mistake, thanks for pointing it out